### PR TITLE
Correct use of random_state for xgboost

### PR DESCRIPTION
**Pull request by lorenjan**
The seed parameter in xgboost was changed to "random_state" to match sklearn's API. This fix corrects the random state behavior and silences a warning which prints on every iteration.

### CODE REVIEW

**Comment by jvdd** [comment]
Thx for submitting this clear PR @lorenjan 

Always pleased to see the community contribute!! :handshake:

**Review by jvdd — APPROVED** [review]
